src/main/java/com/ksm/wstbackoffice/entity/Country.java

```diff
@@ -0,0 +1,12 @@
+package com.ksm.wstbackoffice.entity;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@EqualsAndHashCode
+public class Country {
+    private String title;
```

Comment:
Let make country with the two properties
    private String ISO3166;
    private String name;

Reply:
done